src/js/libcs/Accessors.js

```diff
@@ -168,6 +168,22 @@ Accessor.setField = function(geo, field, value) {
         movepointscr(geo, List.turnIntoCSList([value.value[0], value.value[1], CSNumber.real(1)]), "homog");
     }
 
+    if (field === "x" && geo.kind === "P" && geo.movable && value.ctype === "number") {
+        if (CSNumber._helper.isAlmostZero(geo.homog.value[2])) {
+            console.log("cannot set x component without y component of far points");
```

Comment:
It is CindyScript philosophy (at least I thought so) to silently discard such things. Should you really log this?